### PR TITLE
Testing/favorites

### LINKED ISSUE #12: Start Cypress Testing
- Test user flows
- intercept/stub network requests

### PR DESCRIPTION
**Pull request by CorCanavan**
## Summary:
- This PR adds tests for the user flow of favoriting and deleting a quote card from the Favorites page. 

## Type of Changes:
- [ ] Bug fix
- [ ] Refactor
- [X] New feature

## How was this tested:
Cypress

## What are the relevant tickets:
Resolves #12

### CODE REVIEW

**Comment by VictoriaFC** [comment]
Solid testing! Thanks for taking care of testing the favorites page!

File: src/dailyquote/DailyQuote.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ const DailyQuote = () => {
     fetch("/api/today?key=11ef57ae8191dde524535934c158c4543950e06c")
     .then(response => response.json())
     .then(data => setDailyQuote(data[0]))
-  })
+  }, [])
```
Comment (abodnar1) [inline src/dailyquote/DailyQuote.tsx:17]: nice catch on this!

File: src/Form/Form.tsx [hunk 2]
```diff
@@ -1,10 +1,10 @@
 import React, { useEffect, useState } from "react";
 import "./Form.css";
+import { QuoteInterface } from "../interfaces";
 
 interface formProps {
-	setAllQuotes: Function;
+	setAllQuotes: React.Dispatch<React.SetStateAction<QuoteInterface[]>>;
```
Comment (tfoxcollis) [inline src/Form/Form.tsx:6]: Glad this is working now!

**Review by tfoxcollis — APPROVED** [review]
Thanks for doing this!

**Pull request merged**